app.js

```diff
@@ -1,6 +1,27 @@
-function Book(title, author, pages, read) {
+
+let myLibrary = [];
+let operation = null;
+const formBtn = document.querySelector('#new-btn');
+const submitBook = document.querySelector('.submit-button');
+const form = document.querySelector('.input-form');
+
+// for the local storage
+let storedBooks = JSON.parse(localStorage.getItem("myLibrary"));
+if(storedBooks)
+  {
+    myLibrary = [...storedBooks];
```

Comment:
No need to use spread operator here as what we are always returning the entire content of stored books.

Reply:
We store and call the books from an array called myLibrary. This array is not stored in the local storage. 

When we open the app for the first time, we want to pass all the contents of storedBooks into the myLibrary array to update it and show the stored elements in our local storage.